opp_gui/src/widgets/tool_path_planner_widget.cpp

```diff
@@ -192,7 +193,7 @@ void ToolPathPlannerWidget::loadMeshFromResource()
     std::smatch match;
     if (std::regex_search(filename, match, rgx))
     {
-      mesh_resource_ = "file://" + filename;
+      mesh_resource_ = filename;
```

Comment:
Why remove the file URI? Is it needed by other elements of the code besides the visualization_msgs/Marker (which is being replaced in this PR by the colorized mesh display)?

Reply:
The mesh resource was only being used to get the mesh resource for the marker object. However, since we now are using the pcl loading function, we need to use an absolute path.